src/Response/SearchCityByPostCode.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\MeestBundle\Response;
+
+use Answear\MeestBundle\Response\DTO\CityDTO;
+
+class SearchCityByPostCode implements ResponseInterface
+{
+    /**
+     * @var CityDTO
```

Comment:
Shouldn't `null` be allowed for when there's no matching city?

Reply:
Also in Poland it's possible to have multiple cities assigned to a single postal code. Is it also possible here?

Reply:
I believe this should be `CityDTO[]`

Reply:
According to documentation it cannot be an array, also I believe its not real life scenario given how postal codes are assigned.